rmw_connextdds_common/src/common/rmw_context.cpp

```diff
@@ -471,6 +471,7 @@ rmw_context_impl_s::configure_security(DDS_DomainParticipantQos * const qos)
   {
     RMW_CONNEXT_LOG_ERROR_A_SET(
       "failed to assert DDS property: '%s' = '%s'",
+      DDS_SECURITY_IDENTITY_CERTIFICATE_PROPERTY,
```

Comment:
Shouldn't this be `DDS_SECURITY_PRIVATE_KEY_PROPERTY`?

Reply:
Ah right, i picked up from wrong line.